richtext/features/org.polarsys.kitalpha.richtext.widget.feature/feature.xml

```diff
@@ -64,4 +64,11 @@
          version="0.0.0"
          unpack="false"/>
 
+   <plugin
```

Comment:
the added preference tab seems to only work for "Configurable" richtext. which is not the case in Capella (yet?) or CCM.
Can you create an additional feature to contains the plugin, rather than the main richtext.widget to make it optional and not installed by default.